python/cudf/cudf/core/dataframe.py

```diff
@@ -6019,11 +6019,51 @@ def select_dtypes(self, include=None, exclude=None):
         return df
 
     @ioutils.doc_to_parquet()
-    def to_parquet(self, path, *args, **kwargs):
+    def to_parquet(
+        self,
+        path,
+        engine="cudf",
+        compression="snappy",
+        index=None,
+        partition_cols=None,
+        partition_file_name=None,
+        partition_offsets=None,
+        statistics="ROWGROUP",
+        metadata_file_path=None,
+        int96_timestamps=False,
+        row_group_size_bytes=None,
+        row_group_size_rows=None,
+        max_page_size_bytes=None,
+        max_page_size_rows=None,
+        storage_options=None,
+        return_metadata=False,
+        *args,
+        **kwargs,
```

Comment:
Can you give a little more context on what arguments might fall into this category? You mentioned it might vary by engine? Do we need both `*args` and `**kwargs` here?

Reply:
Had to keep those because other engines by themselves support params like `kwargs`: https://arrow.apache.org/docs/python/generated/pyarrow.parquet.write_to_dataset.html
`args` is likely due to dask upstream API passing it.